src/ocaml_lsp.ml

```diff
@@ -1,65 +1,105 @@
 open! Import
 
+module Experimental_capabilities = struct
+  type t =
+    { interfaceSpecificLangId : bool
+    ; handleSwitchImplIntf : bool
+    ; handleInferIntf : bool
+    ; handleTypedHoles : bool
+    }
```

Comment:
This module is a good idea. Eventually, we need to move it to ocamllsp and then just use it here.